web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -699,6 +699,28 @@ export const formatters: Formatters = {
     desc: 'SSO Test Flow Login Failed',
     format: ({ error }) => `SSO Test flow: user login failed [${error}]`,
   },
+  [eventCodes.USER_HEADLESS_LOGIN_REQUESTED]: {
+    type: 'user.login',
+    desc: 'Headless Login Requested',
+    format: ({ user }) => `Headless login was requested for user [${user}]`,
+  },
+  [eventCodes.USER_HEADLESS_LOGIN_APPROVED]: {
+    type: 'user.login',
+    desc: 'Headless Login Approved',
+    format: ({ user }) =>
+      `User [${user}] successfully approved headless login request`,
+  },
+  [eventCodes.USER_HEADLESS_LOGIN_APPROVEDFAILURE]: {
+    type: 'user.login',
+    desc: 'Headless Login Failed',
+    format: ({ user }) =>
+      `User [${user}] tried approve headless login request, but got an error`,
```

Comment:
```suggestion
      `User [${user}] tried to approve headless login request, but got an error`,
```


Also, is it possible to print the error? I think we do it for the other failure events.

Reply:
Added printing out of the error.